lib/rubocop/comment_config.rb

```diff
@@ -31,22 +31,21 @@ def extra_enabled_comments
       )
     end
 
-    def comment_only_line?(line_number)
-      non_comment_token_line_numbers.none? do |non_comment_line_number|
-        non_comment_line_number == line_number
-      end
+    # Checks if line contains only comment
+    def line_with_comment_only?(line_number)
```

Comment:
What's the reason to rename this?

Reply:
I think it was misunderstanding what method do. First time I've read `comment_only_line` as it's comments only current line. Like so `def foo # rubocop:disable all`. But it really means another thing: line has only comment.

Reply:
Well, it was meant to mean `comment-only line`, which is pretty much the same. If that's the only reason I'd rather go back to the original name to reduce the scope of the change to whatever's really meaningful.

Reply:
ok. I'll return it back

Reply:
it's done